### ISSUE TITLE
Single LButton Long Press

### ISSUE THREAD

**Issue by actforjason** [RamonUnch/AltSnap#139]
AHK
```
~LButton::
        MouseGetPos, x1, y1
        KeyWait, LButton, T0.35
        MouseGetPos, x2, y2
        if (ErrorLevel And x2 = x1 And y2 = y1){
        ; KeyWait, RButton, D
        ; KeyWait, RButton
        ; Sleep 100
            SendInput, {LWin Down}{LButton Up}
            ; Sleep, 3000
            SendInput, {LButton Down}
            KeyWait, LButton
            sendinput, {LWin Up}
        } 
        ; Else SendInput, {LButton}
        ; LButton Up:: SendInput, {LWin Up}
        Return
```

**Comment by RamonUnch** [comment]
It is technically possible on AltSanp as like you asked before to use L/RButton as a hot-click, However, I do not recommend it as it will not let you click.

It is a good idea to trigger AltSnap after a small delay, I would not use it myself, but I do understand the appeal and it would be very useful for some people.
I will see on how to implement this natively.

**Comment by RamonUnch** [comment]
Here is a version of AltSnap that adds the option **Move windows with a long left-click**

[AltSnap1.49bin.zip](https://github.com/RamonUnch/AltSnap/files/7831560/AltSnap1.49bin.zip)
[AltSnap1.49bin_x64.zip](https://github.com/RamonUnch/AltSnap/files/7831562/AltSnap1.49bin_x64.zip)
[AltSnap_installers.zip](https://github.com/RamonUnch/AltSnap/files/7831564/AltSnap_installers.zip)

The option is limited to the move action and the left click.
Like in your AHK script, a mouse up event needs to be generated, so a click is performed before the window is grabbed.
Because of this doing this with the right click would pop-up so much the context menu it would be awkward and unusable.
There is no way not to generate a click befre grabbing the window.
The option can also be edited via the .ini file:

```
LongClickMove=0
; Delay in ms before turning a long click in a move.
; This is a way to use AltSnap without keyboard.
; Set to 0 to disable (default)
; Set to 1 to enable and long click delay will be double click delay.
; Set to any positive value in ms as long click delay eg: 300
```

**Comment by RamonUnch** [comment]
Included in the 1.49 release
